README.md

```diff
@@ -4,6 +4,12 @@ A simple Django middleware for submitting timings and exceptions to Datadog.
 
 ## Installation
 
+Install `dogapi`:
+
+```bash
+pip install dogapi
+```
```

Comment:
This information is unnecessary, as the dependent package is [installed automatically](https://github.com/conorbranagan/django-datadog/blob/master/setup.py#L24-L26) with this package.

Also, `dogapi` is [now deprecated](https://github.com/DataDog/dogapi#deprecated), anyway.

I think this PR can be closed.